plugins/bigquery/dbt/adapters/bigquery/connections.py

```diff
@@ -425,5 +447,8 @@ def count_error(self, error):
 
 
 def _is_retryable(error):
-    """Return true for 500 level (retryable) errors."""
-    return isinstance(error, google.cloud.exceptions.ServerError)
+    """Return true for errors that are unlikely to occur again if retried."""
+    for error_type in RETRYABLE_ERRORS:
+        if isinstance(error, error_type):
```

Comment:
See above, I think this can just be `if isinstance(error, RETRYABLE_ERRORS)`

Reply:
Done.